PurchasesCoreSwift/FoundationExtensions/Dictionary+Extensions.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Dictionary+Extensions.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 7/21/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+extension Dictionary {
+
+    func removingNSNullValues() -> Dictionary {
+        self.filter { !($0.value is NSNull) }
+    }
+
+}
+
+extension Dictionary {
+
+    /// Merge strategy to use for any duplicate keys.
+    enum MergeStrategy<Value> {
```

Comment:
nitpick: empty line at beginning and end of declarations

Reply:
✅